extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderController.class.php

```diff
@@ -8,6 +8,22 @@ public function getAssets() {
 		$response = $this->getResponse();
 		$response->setFormat( WikiaResponse::FORMAT_JSON );
 		$response->setVal( 'css', AssetsManager::getInstance()->getURL( 'portable_infobox_scss' ) );
+
+		$params = $this->getRequest()->getParams();
```

Comment:
what do you think about dividing logic here into two separate functions - one for css assets and one for infobox data? This could be also done while adding sections support to the builder as we plan to refactor data model.